R/utils.R

```diff
@@ -108,5 +115,6 @@ MS2CosineSimilarity <- function(scan1, scan2) {
 #'
 #' @return
 ReplaceNA <- function(column) {
+  # This is a very dangerous function - there's a difference between NA, "NA", and <NA> - which is this meant to handle?
```

Comment:
It's for confidence level 2 and beyond, where the final confidence_source column starts concatenating sources like Ingalls_Standards; MoNA, Metlin. Because there's some pasting going on blank columns end up with a character string of "NA; " which this function just drops. I don't think it's particularly high risk as it doesn't affect normal NAs.